css/index.css

```diff
@@ -104,14 +135,53 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+#box1 {
```

Comment:
use of id selectors to solve this is perfectly fine, as long as you are aware it can also be solved with adding another class and greater specificity